reactioncommerce/reaction
# Fix flash empty cart flashing when registering a user

Resolve issue #3036

This PR fix empty cart showing up while registering new user, The reason for the empty cart showing up is that the cart.items is empty at the that window it tries to register the user. What I did was to create a helper method in checkout.js called cartCount where it checks if the cart.item exists and also check if the length is greater than 0.

#### Test
- Place an order.
- Checkout order.
- Next step will show telling you to register.
- Input details and register.
- Notice that it doesn't show the `empty cart page`
- It takes you to the next step.
